packages/snap-preact-components/src/components/Molecules/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,117 @@
+/** @jsx jsx */
+import { h, Fragment } from 'preact';
+import { useState, useRef } from 'preact/hooks';
+
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+import { observer } from 'mobx-react-lite';
+import deepmerge from 'deepmerge';
+
+import { Icon, IconProps } from '../../Atoms/Icon/Icon';
+import { defined } from '../../../utilities';
+import { Theme, useTheme, CacheProvider } from '../../../providers';
+import { ComponentProps, BreakpointsProps } from '../../../types';
+import { useDisplaySettings } from '../../../hooks/useDisplaySettings';
+
+const CSS = {
+	searchInput: ({ theme, subProps }) =>
+		css({
+			margin: '16px 0 0 0',
+			display: 'flex',
+			flexDirection: 'column',
+			width: '100%',
+			'& .ss__search-input__wrapper': {
+				display: 'flex',
+				alignItems: 'center',
+				width: '100%',
+				height: '35px',
+				border: `1px solid ${theme.colors?.primary || '#ccc'}`,
+				borderRadius: '4px',
+				'& .ss__search-input__wrapper__icon': {
+					display: 'flex',
+					alignItems: 'center',
+					height: '100%',
+					paddingLeft: '0.5rem',
+					paddingRight: '0.5rem',
+				},
+				'& .ss__search-input__wrapper__input': {
+					display: 'flex',
+					alignItems: 'center',
+					width: `calc(100% - 1rem - 2px - ${subProps?.icon?.width || subProps?.icon?.size || '30px'})`,
+					height: '100%',
+					outline: '2px solid transparent',
+					outlineOffset: '2px',
+					paddingRight: '16px',
+					border: '0',
+					boxSizing: 'border-box',
+				},
```

Comment:
To keep things simple and more basic, I think we can get rid of all of these styles except:

```
width: '100%',
outline: 'none',
```

This change would require addressing the border radius overlap problem somehow though...
1. get rid of the border radius
2. add some padding to the wrapper element (padding: 0 4px and box-sizing: border-box)
3. some other clever way